reports/TrialBalance/viewConfig.js

```diff
@@ -30,16 +30,7 @@ export default {
       }
     }
   ],
-  linkFields: [
-    {
-      label: 'Clear Filters',
-      type: 'secondary',
-      action: async report => {
-        await report.getReportData({});
-        report.usedToReRender += 1;
-      }
-    }
```

Comment:
Any reason for removing the _Clear Filter_ from this and General Ledger?

I noticed, prior to this PR this action wasn't implemented, I don't think we should remove it. If it doesn't work we can fix it.

Reply:
Yeah, we'll have to make it work I guess. I've removed them so that we can add them later as required.

Reply:
>  I've removed them so that we can add them later as required.
This doesn't make sense 😅

The problem I see is that someone'll end up implementing a whole other thing to add _Clear Filter_ if we remove it. So since it's already there might as well leave it in. I don't mind pushing commits to fix it.